R/grid_code_paths.R

```diff
@@ -175,13 +177,28 @@ tune_grid_loop_impl <- function(fn_tune_grid_loop_iter,
     # handlers attached to the execution environment of this function
     # by `initialize_catalog()` (#828, #845, #846).
     results <- rlang::with_env(rlang::env_clone(rlang::current_env()), {
+      # Rather than generating them programmatically, write each `foreach()`
+      # call out since `foreach()` `substitute()`s its dots. Note that
+      # doFuture will error when passed `.packages`.
```

Comment:
Paired with Lionel on this. Passing these arguments programmatically seems like more hassle than it's worth. :/